torch/onnx/__init__.py

```diff
@@ -52,3 +53,8 @@ def _run_symbolic_function(*args, **kwargs):
 def _run_symbolic_method(*args, **kwargs):
     from torch.onnx import utils
     return utils._run_symbolic_method(*args, **kwargs)
+
+
+def is_in_onnx_export():
```

Comment:
add docstring to it?

Reply:
Sure!

Reply:
added the doc to onnx/utils.py. We probably should move all the doc to __init__.py